hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/benefits/BenefitInformationController.java

```diff
@@ -67,9 +68,9 @@ public String viewBenefitInfo(ModelMap model, PortletRequest request) {
       model.addAttribute("tab", tab);
       boolean isMadisonUser = !StringUtils.isBlank(userInfo.get("wiscEduSORName"));
       model.addAttribute("isMadisonUser", isMadisonUser);
-      
+      final PortletPreferences preferences = request.getPreferences();
+      model.addAttribute("learnMoreEBenefitGuide", preferences.getValue("ebenefitguidetext", null));
```

Comment:
So, if the user is a Madison user, and the `ebenefitguidetext` portlet preference is not set, then `${learnMoreEBenefitGuide}` is `null`? Which prints to the screen as, what, "null"? Should the default value for the preference here be `""` rather than `null`?